stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -424,10 +434,18 @@ ::util::Status GetAllEntries(
     std::unique_ptr<::tdi::TableData> table_data;
     RETURN_IF_TDI_ERROR(table->keyAllocate(&table_key));
     RETURN_IF_TDI_ERROR(table->dataAllocate(&table_data));
-    RETURN_IF_TDI_ERROR(table->entryGetFirst(
-        *tdi_session, tdi_dev_target,
+    tdi_status = table->entryGetFirst(
```

Comment:
Instead of declaring `tdi_status` at the top of the function and initializing it here, it is better C++ style to specify:
```
auto tdi_status = table->entryGetFirst( ... );
```